modAionImpl/src/org/aion/zero/impl/config/CfgAion.java

```diff
@@ -136,6 +139,51 @@ private void closeFileInputStream(final FileInputStream fis) {
         }
     }
 
+    public void setForkProperties() {
+        setForkProperties("mainnet");
+    }
+
+    public void setForkProperties(String networkName) {
+        Properties properties = new Properties();
+
+        // old kernel doesn't support the fork feature.
+        if (networkName.equals("config")) {
+            return;
+        }
+
+        FileInputStream fis = null;
+        try {
```

Comment:
optional: can use try-with-resources; i.e. `try (FileInputStream fis = new FileInputStream( ... )) { ... } ` and omit the `finally { /* close */ }`